pygmt/src/grdlandmask.py

```diff
@@ -20,6 +20,7 @@
     I="spacing",
     R="region",
     r="registration",
+    V="verbose",
```

Comment:
Sort by [ASCIIbetical](https://en.wikipedia.org/wiki/ASCII#Character_order) order so that capital letters come first :)

```suggestion
    V="verbose",
    r="registration",
```